build_modules/pubspec.yaml

```diff
@@ -1,13 +1,13 @@
 name: build_modules
-version: 5.0.9-wip
+version: 5.0.9
 description: >-
   Builders to analyze and split Dart code into individually compilable modules
   based on imports.
 repository: https://github.com/dart-lang/build/tree/master/build_modules
 resolution: workspace
 
 environment:
-  sdk: '>=3.5.0-259.0.dev <3.6.0'
+  sdk: '>=3.5.0-259.0.dev <3.7.0'
```

Comment:
Is there any reason not to change the lower bound to `>=3.5.0`?

Reply:
Well, 3.5 isn't actually out yet.

Reply:
At a minimum our "pubspec" SDK jobs would fail.

Reply:
Ah, surprising that we're prepping for 3.6 without launching 3.5...

LGTM.

Reply:
Yeah it happens when we cut the branch that will be 3.5 stable. That way we don't have misleading -dev tags that make people put in bad constraints.